cli/sub_command.go

```diff
@@ -64,7 +64,23 @@ type subCmd struct {
 
 func configureSubCommand(app commandHost) {
 	c := &subCmd{}
+
+	subHelp := `
+	Jetstream will be activated when related options like --stream, --durabled or --ack are supplied.
```

Comment:
technically you can also do `nats sub <some subject that is in a stream> --all` (or --last, --new, --since, --last-per-subject) to activate jetstream.  It will go find the right stream on its own.

Reply:
Hm, can't find that code. But it works. Here by dragons ....

Reply:
Ah, this is an inherent JS feature! Wasn't aware of it.

Reply:
https://github.com/nats-io/natscli/blob/3a7c770528117163dd61a94cc4c903792b66753d/cli/sub_command.go#L169

Reply:
The "magic" I was referring to was that I can subscribe to a subject from Jetstream without providing a stream and the server will resolve the stream implicitly.